.buildkite/custom-tests.json

```diff
@@ -0,0 +1,9 @@
+{
+  "tests": [
+    {
+      "test_name": "build-mshv",
+      "command": "cargo build --release --no-default-features --features mshv",
```

Comment:
You should also run clippy on the mshv feature because it is currently running with `--all-features`: https://github.com/rust-vmm/rust-vmm-ci/blob/ae7db2d98a071f52de3d60af9c937204b1f087a4/.buildkite/test_description.json#L41

You'd just need to replace `--all-features` with `--feature mshv`

Reply:
Good suggestion. I've added a new test for that.